3_javascript/javascript1/week2/homework javascript1 week2/studentm.js

```diff
@@ -0,0 +1,32 @@
+//Student manager && addStudentToClass function
+
+let class07Students = [];
+function addStudentToClass(studentName) {
+  if (studentName === "") {
+    console.log("Kindly add more students to the class!");
+  }
+  for (let i = 0; i < class07Students.length; i++) {
+    if (class07Students[i] === studentName) {
+      console.log(`${studentName} is already in the class`);
+      return;
+    }
+  }
+  if (class07Students.length < 7) {
```

Comment:
The structure of this is awesome! 💪 However, there are a few logical bugs, can you spot them? 🐛 For an example, when the class is at maximum capacity (6), what happens if you add another student? What happens when you add the Queen after the class is full? 👑